src/style.css

```
@@ -1,3 +1,93 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.li {
+  height: 100%;
+}
+
+.nav {
+  display: flex;
+  height: 60px;
+  margin: 0;
+  padding: 0;
+  margin-right: 50px;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-weight: 500;
+  height: 100%;
+  line-height: 60px;
+}
+
+.nav_item {
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  padding: 0;
+  margin: 0;
+}
+
+.logo_img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  margin: 10px 50px;
+}
```

it's redundant because you're using `flex` with the `space-between` property in your header, as well as zero padding.
Try using horizontal padding in your header

```suggestion
.logo_img {
  width: 40px;
  height: 40px;
}
```